src/awscli_login/__init__.py

```diff
@@ -41,34 +41,71 @@ class Login(BasicCommand):
                    ' for authentication.')
     SYNOPSIS = ('aws login [<Arg> ...]')
 
+    # tests/util.py:login_cli_args defaults must match this table
     ARG_TABLE = [
-        {'name': 'entity-id', 'help_text': 'Entity ID of the IdP'},
+        # Ordering matches order in README.rst
+        # Basic Properites (can be set interactively)
         {
             'name': 'ecp-endpoint-url',
+            'default': None,
             'help_text': 'ECP endpoint URL of the IdP'
         },
-        {'name': 'username', 'help_text': 'Username to use on login to IdP'},
-        {'name': 'password', 'help_text': 'Password to use on login to IdP'},
+        {
+            'name': 'username',
+            'default': None,
+            'help_text': 'Username to use on login to IdP'
+        },
+        {
+            'name': 'password',
+            'default': None,
+            'help_text': 'Password to use on login to IdP'
+        },
+        {
+            'name': 'factor',
+            'default': None,
+            'help_text': 'The Duo factor to use on login'
+        },
+        {
+            'name': 'passcode',
+            'default': None,
+            'help_text': 'A Duo passcode'
+        },
         {
             'name': 'role-arn',
+            'default': None,
             'help_text': 'The Role ARN to select. '
                          'If the IdP returns a single Role it is autoselected.'
         },
-        {'name': 'factor', 'help_text': 'The Duo factor to use on login'},
-        {'name': 'passcode', 'help_text': 'A Duo passcode'},
+        # Advancded Properites (can NOT be set interactively)
+        {
+            'name': 'disable-refresh',
+            'default': None,
+            'cli_type_name': 'boolean',
+            'help_text': 'Disables automatic refresh of tokens'
+        },
         {
             'name': 'refresh',
-            'default': 0,
+            'default': None,
             'cli_type_name': 'integer',
             'help_text': 'How often in seconds to refresh the STS credentials'
         },
         {
-            'name': 'verbose',
-            'action': 'count',
-            'default': 0,
+            'name': 'duration',
+            'default': None,
             'cli_type_name': 'integer',
-            'help_text': 'Display verbose output'
+            'help_text': 'Time in seconds that the token will last'
         },
+        {
+            'name': 'http_header_factor',
+            'default': None,
+            'help_text': 'Name of HTTP Header to store the user\'s factor in'
+        },
+        {
+            'name': 'http_header_passcode',
+            'default': None,
+            'help_text': 'Name of HTTP Header to store the user\'s passcode in'
```

Comment:
This is nitpicky, but 'Name of HTTP header in which to store the user\'s password' is cleaner.